ConTabsDemo-DotNetCore/ConTabsDemo-DotNetCore.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
This worries me, as it may cause a loss of backwards compatibility. Is it necessary?

Reply:
Thanks so much for the feedback! Your comments are very welcomed, and the requests of course very reasonable. Late night / lazy programming did create a lot of noise. 

I'll submit a new PR with the following changes:
1) The Resharper noise is gone.
2) The target framework was reverted to 1.1. Frankly, this only happened because I didn't have 1.1 installed on my machine so. It was not necessary.